osu.Game.Rulesets.Osu/Mods/OsuModFlashlight.cs

```diff
@@ -56,7 +56,7 @@ protected override bool OnMouseMove(MouseMoveEvent e)
                 var destination = e.MousePosition;
 
                 FlashlightPosition = Interpolation.ValueAt(
-                    Math.Clamp(Clock.ElapsedFrameTime, 0, follow_delay), position, destination, 0, follow_delay, Easing.Out);
+                    Math.Clamp(Math.Abs(Clock.ElapsedFrameTime), 0, follow_delay), position, destination, 0, follow_delay, Easing.Out);
```

Comment:
```suggestion
                    Math.Min(Math.Abs(Clock.ElapsedFrameTime), follow_delay), position, destination, 0, follow_delay, Easing.Out);
```

Reply:
Hm, I guess that's a correct optimisation to make, although it's a bit of a clever one and clever code isn't always the best code... @peppy thoughts?

Reply:
i'm fine either way. makes sense that the lower clamp isn't required due to abs.